packages/@yoda/network/mock.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+var flora = require('@yoda/flora')
+var agent = new flora.Agent('unix:/var/run/flora.sock#net_manager')
+var current_wifi_state = "DISCONNECTED"
+
+function handleWifiCommand(msg) {
+  switch (msg.command) {
+    case "CONNECT":
+      current_wifi_state = "CONNECTED"
+      return { retCode: 0, msg: [] }
+
+    case "DISCONNECT":
+      current_wifi_state = "DISCONNECTED"
+      return { retCode: 0, msg: [] }
+
+    case "START_SCAN":
+    case "STOP_SCAN":
+      return { retCode: 0, msg: [] }
+
+    case "GET_WIFILIST":
+      var result = {
+        "wifilist": [
+          {"ssid": "ROKID.TC", "signal": -50},
+          {"ssid": "ROKID.GUEST", "signal": -50}
+        ]
+      }
+      return { retCode: 0, msg: [JSON.stringify(result)] }
+  }
+}
+
+agent.declareMethod('command', (msg) => {
+  var msg = JSON.parse(msg[0])
+  console.log(msg)
+  if (msg.device === "WIFI") {
+    return handleWifiCommand(msg)
+  }
+})
+
+agent.start()
+
+setInterval(() => {
+  var msg = {
+    wifi: {
+      state: current_wifi_state
+    }
+  }
+
+  agent.post('network.status', [ JSON.stringify(msg) ], flora.MSGTYPE_INSTANT)
+}, 3000)
```

Comment:
Is this used for testing? Add a test case under our `test/@yoda` would be better :)